lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -23,6 +23,11 @@ def add(*args, &block)
     true
   end
 
+  def reopen(logdev = nil)
+    loggers.each { |l| l.reopen(logdev) }
```

Comment:
I think this should probably just `raise NotImplementedError` for now if we can't agree on a clever way to do this.
Then the spec may just need to redefine `$api_log` rather than reopening it.

Alternatively this could reopen just the first logger, which should be okay for all use cases outside of containers, but maybe not what was expected to happen.

Then if we really need this to work better maybe if the parameter is an array we just close out the existing loggers and replace them with new loggers with the list of logdevs.

As of now the api logging spec is the only place in the entire app that we call `Logger#reopen` so there's not a lot of risk here.

I'm leaning towards raise if the fix to the spec is easy otherwise just change the first logger, which, in the worst case, could overwrite our STDOUT logger and end up logging twice to the file ...